-- cmake/3rdPartyTools.cmake.orig	2025-05-31 18:32:29.248332000 +0000
+++ cmake/3rdPartyTools.cmake	2025-05-31 18:34:10.499384292 +0000
@@ -11,7 +11,8 @@
 zlib
 libbz2
 kmmd
-libm)
+libm
+nccl)
 
 set(3RDPARTY_TOOL_USES
 "for fundamental linear algebra calculations"
@@ -21,7 +22,8 @@
 "for various compression and decompression tasks"
 "for various compression and decompression tasks"
 "Machine-learning molecular dynamics"
-"for fundamental math routines if they are not contained in the C library")
+"for fundamental math routines if they are not contained in the C library"
+"for interGPU communications")
 
 # Logic to disable tools
 set(3RDPARTY_SUBDIRS "")
